doc/man1/openssl-version.pod.in

```diff
@@ -77,6 +78,11 @@ The random number generator source settings.
 
 The OpenSSL CPU settings info.
 
+=item B<-w>
+
+The OpenSSL WININSTALLCONTEXT build time variable, if set.
```

Comment:
Most of this PR mention OSSL_WINCTX, not WININSTALLCONTEXT.
I do not see any more WININSTALLCONTEXT reference.

Reply:
yeah, thats outdated, will fix